### PR TITLE
[Magic Network Monitoring] New dash network analytics for MT and MWAN

### PR DESCRIPTION
**Pull request by marciocloudflare**
Addresses PCX-5945

### CODE REVIEW

File: content/magic-wan/how-to/view-analytics.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 
-# View Network Analytics
+{{<render file="_network-analytics.md" withParameters="Magic WAN">}}
 
-Cloudflare’s Network Analytics provides near real-time visibility into network and transport layer traffic patterns and DDoS attacks which can help troubleshoot IP traffic issues. You can also use Network Analytics to view information about the traffic that leaves Cloudflare's global network by reviewing ingress and egress tunnel traffic over a specific amount of time. 
+## Query analytics with GraphQL
 
-For more information, refer to [Querying Magic Transit Tunnel Bandwidth Analytics with GraphQL](/analytics/graphql-api/tutorials/querying-magic-transit-tunnel-bandwidth-analytics/).
\ No newline at end of file
+You can also query analytics data using GraphQL. Refer to For more information, refer to [Querying Magic Transit Tunnel Bandwidth Analytics with GraphQL](/analytics/graphql-api/tutorials/querying-magic-transit-tunnel-bandwidth-analytics/) and [Querying Magic Transit tunnel health check results with GraphQL](/analytics/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/) for more information.
```
Comment (jason-cf) [inline content/magic-wan/how-to/view-analytics.md]: @marciocloudflare "Refer to" and "For more information" here is not correct. Please take a look.

File: content/magic-wan/_partials/_network-analytics.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- Source and destination IP addresses, ports, and protocols of tunnel traffic.
+- Samples of all GRE or IPSec tunnel traffic entering or leaving Cloudflare’s network.
+- Mitigations applied (DDoS, Magic Firewall, etc.) to traffic entering Cloudflare.
+
+## Access tunnel analytics
+
+1. Go to the [Cloudflare dashboard](https://dash.cloudflare.com/login) and select your account.
+2. Select **Analytics & Logs** > **Network Analytics**.
+3. In the **All Traffic** tab, scroll to **Top Insights** to access the **Source tunnel** and **Destination tunnel** panels. Here you can examine information from your top tunnels by traffic volume.
+4. You can also apply filters to adjust the scope of information displayed. Scroll to **All traffic** > **Add filter**.
```
Comment (jason-cf) [inline content/magic-wan/_partials/_network-analytics.md:25]: capital "All Traffic" ?

File: content/magic-wan/_partials/_network-analytics.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+5. In the **New filter** popover, choose what type of data you want to display from the left dropdown menu, an operator from the middle dropdown menu, and an action from the right dropdown menu. For example:
+
+    ```txt
+    <DESTINATION_TUNNELS> | _equals_ | <NAME_OF_YOUR_TUNNEL>
+    ```
+
+    This lets you examine traffic from specific Source tunnels and / or Destination tunnels.
```
Comment (jason-cf) [inline content/magic-wan/_partials/_network-analytics.md]:
```suggestion
    This lets you examine traffic from specific Source tunnels and/or Destination tunnels.
```

**Review by jason-cf — APPROVED** [review]

File: content/magic-wan/_partials/_network-analytics.md (hunk 2 above)
Comment (marciocloudflare) [reply content/magic-wan/_partials/_network-analytics.md:25]: No, bc this is how the dash is - for now, ate leat

**Pull request merged**